reana-ui/src/pages/workflowDetails/components/filePreview/FilePreview.js

```diff
@@ -151,9 +151,9 @@ function checkConstraints(fileName, size) {
   if (!match) {
     const fileExt = fileName.split(".").pop();
     content = `${fileExt} files cannot be previewed. Please use download.`;
-  } else if (size > FILE_SIZE_LIMIT) {
+  } else if (size > sizeLimit) {
     content = `File size is too big to be previewed (limit ${
-      FILE_SIZE_LIMIT / 1024 ** 2
+      sizeLimit / 1024 ** 2
```

Comment:
We may want to round this number to show only one or two decimal digits at max - if `config.filePreviesSizeLimit` is not a multiple of a MiB, the result of the division may show a lot of decimal places.

Reply:
I have updated the PR to show the limit in human-readable format, basically the same as [`_human_readable_bytes`](https://github.com/reanahub/reana-db/blob/02c0e4a93c46bfaa302dd7d84647a92d1673e8c2/reana_db/models.py#L1018) from reana-db